docs/help_for_devs/0001-include-test-cases-files.rst

```diff
@@ -5,81 +5,86 @@ Eox-core test cases
 Context
 -------
 
-In order to have a way to test the eox-core API endpoints manually, but quickly, two json files have been
-created with the GET, POST, PATCH and DELETE requests. The `Eox-core-test.postman_collection.json`_ and
-`Eox-core-test.postman_environment.json`_ files were created with the `Posman`_ application.
+To have a way to test the eox-core API endpoints manually, but quickly, two JSON files have been
+created with GET, POST, PATCH, and DELETE requests. The `Eox-core Test Collection`_ and
+`Eox-core Test Environment`_ files were created for the `Posman`_ application.
 
-.. _eox-core-test.postman_collection.json: ../resources/Eox-core-test.postman_collection.json
-.. _eox-core-test.postman_environment.json: ../resources/Eox-core-test.postman_environment.json
+.. _Eox-core Test Collection: ../resources/Eox-core-test.postman_collection.json
+.. _Eox-core Test Environment: ../resources/Eox-core-test.postman_environment.json
 .. _Posman: https://www.postman.com/
 
 Instructions
 ------------
 
-To test the endpoints it is necessary to have the Open Edx platform running with the eox-core plugin installed.
+To properly test the endpoints, the Open Edx platform must be running with the eox-core plugin installed.
 
 #. Once your environment is set up, create a new admin user: ``tutor dev do createuser --staf --superuser admin <custom-mail> --password <password>``.
```

Comment:
We could use numbers instead of #. It is clearer from the code.
```suggestion
1. Once your environment is set up, create a new admin user: ``tutor dev do createuser --staf --superuser admin <custom-mail> --password <password>``.
```